tests/unit/test_task_remote_cmd.py

```diff
@@ -0,0 +1,51 @@
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Tests for remote initialisation."""
+
+from io import StringIO
+from unittest.mock import patch
+
+from cylc.flow.task_remote_cmd import remote_init
+
+
+@patch('sys.stdout', new_callable=StringIO)
```

Comment:
I'm going to make a PR to your branch for this PR suggesting that you might want to use the built in fixture `capsys` instead of mocking `sys.stdout`.